server_admin/topics/admin-cli.adoc

```diff
@@ -1,1320 +1,1702 @@
-or alternatively you can specify only a client id, which will result in special, so called 'service account' being used. In the first case,
-a password for the specified user has to be used at login. In the latter case there is no user password - only client secret.
-Alternatively, `Signed JWT` can be used.
+When you log in with the Admin CLI, you specify a server endpoint URL and a realm, and then you specify a username. Another option is to specify only a client ID, which results in using a special "service account". When you log in using a username, you must use a password for the specified user. When you log in using a client ID, you only need the client secret, not the user password. You could also use [command]`Signed JWT` instead of the client secret.
 
-The account used for the session needs to have proper permissions in order to be able to invoke Admin REST API operations.
-For example, `realm-admin` role of `realm-management` client allows user to administer the realm within which the user is defined.
+Make sure the account used for the session has the proper permissions to invoke Admin REST API operations. For example, the `realm-admin` role of the `realm-management` client allows the user to administer the realm within which the user is defined.
 
+There are two primary mechanisms for authentication. One mechanism uses [command]`kcadm config credentials` to start an authenticated session.
+[options="nowrap"]
+----
+$ kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user admin --password admin
+----
 
-There are two primary mechanisms for authentication. One is using `kcadm config credentials` to start an authenticated session:
+This approach maintains an authenticated session between the [command]`kcadm` command invocations by saving the obtained access token and the associated refresh token. It may also maintain other secrets in a private configuration file. See <<_working_with_alternative_configurations, next chapter>> for more information on the configuration file.
 
-    $ kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user admin --password admin
+The second approach only authenticates each command invocation for the duration of that invocation. This approach increases the load on the server and the time spent with roundtrips obtaining tokens. The benefit of this approach is not needing to save any tokens between invocations, which means nothing is saved to disk. This mode is used when the [command]`--no-config` argument is specified.
 
-This approach maintains an authenticated session between `kcadm` command invocations by saving the obtained access token, and
-associated refresh token, possibly other secrets as well in a private configuration file. See <<_working_with_alternative_configurations, next chapter>> for more info on configuration file.
+For example, when performing an command, we specify all the information required for authentication.
```

Comment:
The original sentence was on line 92 and read "For example, when performing an operation we specify all the information required for authentication:".

I am curious whether changing "operation" to "command" changes the meaning more than would be preferred. I think it might. Rather than fix the "an" to an "a" perhaps we change back to "operation."

Reply:
To me 'command' and 'operation' mean the same. Go with whatever sounds better.

Reply:
"an command" was changed back to "an operation"